lib/main_common.pm

```diff
@@ -2085,6 +2085,8 @@ sub load_security_tests_crypt_core {
         loadtest "fips/openssl/openssl_fips_alglist";
         loadtest "fips/openssl/openssl_fips_hash";
         loadtest "fips/openssl/openssl_fips_cipher";
+        loadtest "fips/openssl/dirmngr_setup";
```

Comment:
Please add NOTE like: ALWAYS run following 2 test cases in sequence because of the dependencies

Reply:
done, thanks.